elifearticle/parse.py

```diff
@@ -46,34 +46,19 @@ def build_contributors(authors, contrib_type, competing_interests=None):
         if author.get("equal-contrib") == "yes":
             contributor.equal_contrib = True
 
-        # Affiliations, compile text for each
-        department = []
-        institution = []
-        city = []
-        country = []
-
+        # Add contributor affiliations
         if author.get("affiliations"):
             for aff in author.get("affiliations"):
```

Comment:
Could be just `for aff in author.get("affiliations", []):` and get rid of the above `if author.get("affiliations"):` line. 

Avoids a second `dict` lookup and will still skip the below logic if the `"affiliations"` key is not found as it will return the empty `list`.

Reply:
I like it, very smart - I will do!